### PR TITLE
Update Bulgarian localization

### PR DESCRIPTION
**Pull request by rddim**
* Update localization files · notepad-plus-plus/notepad-plus-plus@2258274 (for `Add an option allows to show only 1 entry per found line in search result` · notepad-plus-plus/notepad-plus-plus@08128ee)

### CODE REVIEW

**Comment by rddim** [comment]
@donho 
There is not enough space for the localization of  string `<Item id="6906" name="Search Result window: show only one entry per found line (not applied to RTL mode)"/>`

Image-2022-06-28-18-33-38

I can change it to `Прозорец с резултати от търсенето: показване само на един запис на намерен ред (без режим RTL)` which in english is `Search Result window: show only one entry per found line (without RTL mode)`

**Comment by donho** [comment]
@rddim 
Try to remove the translated word of "Window".

**Comment by rddim** [comment]
> @rddim Try to remove the translated word of "Window".

@donho 
Done, now everythink fits, thanks

Image-2022-06-28-21-01-19

File: PowerEditor/installer/nativeLang/bulgarian.xml [hunk 1]
```diff
@@ -997,6 +997,7 @@
 					<Item id="6903" name="Диалога за търсене остава отворен, след като резултатите бъдат изведени в прозореца"/>
 					<Item id="6904" name="Потвърждение при &quot;Замяна на всичко във всички отворени&quot; документи"/>
 					<Item id="6905" name="Без преместване към следващото повторение при замяна"/>
+					<Item id="6906" name="Резултати от търсене: показване само на един запис на намерен ред (не се прилага за режим RTL)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/bulgarian.xml:1000]: "(not applied to RTL mode)" has been removed in this entry, see: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/059bca3315dbd71fbf235c8bc87eaacbd87a3cec
Sorry for the inconvenience.

**Review by donho — CHANGES_REQUESTED** [review]
Please remove the translation of "(not applied to RTL mode)".

**Comment by rddim** [comment]
@donho 

Done. Sorry for the late reply, I was away from my PC.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**